src/Schema/AbstractSchemaManager.php

```diff
@@ -33,24 +33,10 @@
 abstract class AbstractSchemaManager
 {
     /**
-     * Holds instance of the Doctrine connection for this schema manager.
+     * @param T $_platform
      */
-    protected Connection $_conn;
-
-    /**
-     * Holds instance of the database platform used for this schema manager.
-     *
-     * @var T
-     */
-    protected AbstractPlatform $_platform;
-
-    /**
-     * @param T $platform
-     */
-    public function __construct(Connection $connection, AbstractPlatform $platform)
+    public function __construct(protected Connection $_conn, protected AbstractPlatform $_platform)
```

Comment:
This change is a bit unfortunate because the constructor parameter names now have that ugly underscore that we wanted to get rid of.

I'd either revert the use of CPP here or rename the properties to match the parameter names instead of the other way round.

Reply:
CPP is a good reason to get rid at least of some ugly underscores in protected property names.